protocol/v2/ssv/runner/committee.go

```diff
@@ -410,6 +420,11 @@ func (cr *CommitteeRunner) ProcessPostConsensus(ctx context.Context, logger *zap
 
 			vlogger.Debug("🧩 reconstructed partial signatures committee",
 				zap.Uint64s("signers", getPostConsensusCommitteeSigners(cr.BaseRunner.State, root)))
+
+			if cr.doppelgangerHandler.ValidatorStatus(validator) == doppelganger.SigningDisabled {
+				cr.doppelgangerHandler.MarkAsSafe(validator)
+			}
+
```

Comment:
not sure about it, but should we do it as well if the only duty that we have quorum on is sync committte? since we send sync committes even when we're in doppleganger state.

so we might want to move this downwards into the place where we know the role is attestaions.

@moshe-blox @MatusKysel @olegshmuelov wdyt?

Reply:
good catch!
https://github.com/ssvlabs/ssv/pull/2036/commits/7d978a71f61dcd7cd917e8b3d0d86578ff926181